src/protos/phy.cc

```diff
@@ -39,9 +39,7 @@ struct PhyImpl {
     uint32_t snap;
     float late;
 
-#ifdef HAVE_DAQ
```

Comment:
This is the opposite of what is desired.  See the commit in my PR for fixing the build without libdaq.

Reply:
Ya, this was hack'n slashed to build without any real thought